index.html

```diff
@@ -22,7 +22,7 @@
   <body>
     <header>
       <h1>The Algorithmist</h1>
-      <h3>Advanced algorithms plainly explained</h3>
```

Comment:
I'm going to argue that "plainly" is different from "simply" - it just means "clearly", not "dumbed down". I prefer the original subtitle for this reason and its cadence, personally.

Reply:
I think it's easy to misinterpret "plainly" as "simply":
https://www.google.com/search?client=ubuntu&channel=fs&q=define+plainly&ie=utf-8&oe=utf-8

To maintain the cadence, is there another adjective you like?

Reply:
Is your opposition to "simply" because it seems dumbed down? Aside from that, don't we want the explanations to be clear and straightforward? Like, isn't that the point?

Maybe "thoroughly" instead of "plainly"?

Reply:
> Is your opposition to "simply" because it seems dumbed down? Aside from that, don't we want the explanations to be clear and straightforward? Like, isn't that the point?
I don't want us to sound condescending if anyone reads the post and can't figure out what we're trying to say.
 
> Maybe "thoroughly" instead of "plainly"?
Up to you. I think "throughly" is better than "plainly".

Reply:
I'm good with that. Can you make the change, since I'm on mobile at the moment?